devtools/test_dashboard/index.html

```diff
@@ -19,7 +19,7 @@
   </div>
   <div id="snapshot"></div>
 
-  <script src="../../dist/extras/mathjax/MathJax.js?config=TeX-AMS-MML_SVG"></script>
+  <script src="../../vendor/extras/mathjax/MathJax.js?config=TeX-AMS-MML_SVG"></script>
```

Comment:
Can we drop the `extras` here? Just `vendor/mathjax`?

Reply:
We can. But we are going to remove this folder right after to upgrade to 2.5.1 then 2.7.x.
Do you still prefer moving those?